tests/test_main.py

```diff
@@ -64,6 +64,14 @@ def folder(self):
         return join("tests", "fixtures")
 
     def test_pipelines(self, configuration, folder):
+        configuration["HAPI_countries"] = [
+            "AFG",
+            "BFA",
+            "MLI",
+            "NGA",
+            "TCD",
+            "YEM",
+        ]
```

Comment:
I included this to try to control which countries are run in the scraper, but it's not having the effect I hoped for.  The tests fail looking for data for CMR. 
 @mcarans is there a way to limit the countries I'm running like in the covid viz scraper?

Reply:
This will require making a change to HDX Python Scraper to allow limiting the countries